https://issues.redhat.com/browse/ACM-28563  Update search api with 'offset' and 'orderBy' parameters (#8604)
open-cluster-management/rhacm-docs: release_notes/acm_new.adoc

- The console now supports {hcp} upgrade paths and displays a channel modal to manage channels for hosted clusters, with available channels for upgrade in the `ManagedClusterInfo` resource.

> - Configure pagination for your search results with the `offset` parameter. For more information, see link:../apis/searchquery.json.adoc#search-query-api[Search query API].

Reviewer: ```suggestion
- Configure pagination for your search results with the `offset` and `orderBy` parameters. For more information, see link:../apis/searchquery.json.adoc#search-query-api[Search query API].
```

Author: Actually in this context it would be `offset`, `limit`, and `orderBy`

The `limit` option existed before.

> - Configure pagination for your search results with the `offset` and `orderBy` parameters. For more information, see link:../apis/searchquery.json.adoc#search-query-api[Search query API].

Reviewer: ```suggestion
- Configure pagination for your search results with the `limit`, `offset`, and `orderBy` parameters. For more information, see link:../apis/searchquery.json.adoc#search-query-api[Search query API].
```

> - Configure pagination for your search results with the `limit`, `offset`, and `orderBy` parameters. For more information, see link:../apis/searchquery.json.adoc#search-query-api[Search query API].

Reviewer: ```suggestion
- Configure pagination for your search results with the `offset` and `orderBy` parameters in the search API. For more information, see link:../apis/searchquery.json.adoc#search-query-api[Search query API].
```

> - Configure pagination for your search results with the `offset` and `orderBy` parameters in the search API. For more information, see link:../apis/searchquery.json.adoc#search-query-api[Search query API].

Reviewer: ```suggestion
- Manage large search datasets by configuring pagination with the `limit`, `offset`, and `orderBy` parameters in the search API. For more information, see link:../apis/searchquery.json.adoc#search-query-api[Search query API].
```
